D/DecFP/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+using BinaryBuilder
+
+name = "DecFP"
+version = v"2.0U2" # 2.0 Update 2
+
+# Collection of sources required to build DecFP
+sources = [
+    ArchiveSource("https://www.netlib.org/misc/intel/IntelRDFPMathLib20U2.tar.gz",
+                  "93c0c78e0989df88f8540bf38d6743734804cef1e40706fd8fe5c6a03f79e173"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/IntelRDFPMathLib20U2
+patch --binary -p1 < $WORKSPACE/srcdir/patches/decfp.patch
+cd LIBRARY
+if [[ $nbits == 64 ]]; then
+     _HOST_ARCH=x86_64
+else
+     _HOST_ARCH=x86
+fi
+if [[ $target == *-w64-* ]]; then
+    _HOST_OS="Windows_NT"
+    objext="obj"
+elif [[ $target == *-freebsd* ]]; then
+    _HOST_OS="FreeBSD"
+    objext="o"
+else
+    _HOST_OS="Linux"
+    objext="o"
+fi
```

Comment:
So macOS is under
```
_HOST_OS="Linux"
```
?

Reply:
I have tested this on macOS using _HOST_OS="Linux" and it works.  Looking at it now, I see there is a Darwin target, so I'll test this.

Reply:
It doesn't seem it worked :confused:  Let's go back to `_HOST_OS="Linux"`?  I don't know how this variable is used, maybe it isn't too important in our case.

Reply:
That variable is used in the Makefiles to determine which commands to run and which options to pass.  In this case it failed on the line:
```
ar rv -o libbid.a files.o ...
```
The `-o` shouldn't be there.  Back to `_HOST_OS="Linux"`